src/schematics/deploy/functions-templates.ts

```diff
@@ -0,0 +1,29 @@
+export const defaultPackage = `{
+  "name": "functions",
+  "description": "Angular Universal Application",
+  "scripts": {
+    "lint": "",
+    "serve": "firebase serve --only functions",
+    "shell": "firebase functions:shell",
+    "start": "npm run shell",
+    "deploy": "firebase deploy --only functions",
+    "logs": "firebase functions:log"
+  },
+  "engines": {
+    "node": "8"
+  },
+  "dependencies": {
```

Comment:
Is there a way we could allow configuration of webpack externals here? E.g, libs that they don't want bundled but use in SSR